README.md

```diff
@@ -8,13 +8,13 @@ Progress indicators and spinners for React Native using ReactART.
 
 `$ npm install react-native-progress --save`
 
-### ReactART based components
+### React Native SVG based components
```

Comment:
There is still a reference to ReactART above :point_up:  on line 3 https://github.com/oblador/react-native-progress/pull/229/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3

More importantly: thank you for proposing this PR!

Reply:
Thanks - Fixed it!